### DISCUSSION TITLE
Wiki js container failing

### DISCUSSION THREAD

**Discussion by Kill3r1322** [tteck/Proxmox#4019] [Help Needed] [answered]
I am trying to deploy wiki js lxc and I am getting error below:
`Saving to: ‘wiki-js.tar.gz’

wiki-js.tar.gz                           100%[================================================================================>]  77.68M  3.04MB/s    in 26s     

2024-10-31 14:42:35 (3.01 MB/s) - ‘wiki-js.tar.gz’ saved [81452882/81452882]

npm error code 6
npm error path /opt/wikijs/node_modules/sqlite3
npm error command failed
npm error command sh -c node-pre-gyp install --fallback-to-build
npm error node-pre-gyp info it worked if it ends with ok
npm error node-pre-gyp info using node-pre-gyp@1.0.11
npm error node-pre-gyp info using node@20.18.0 | linux | x64
npm error node-pre-gyp info check checked for "/opt/wikijs/node_modules/sqlite3/lib/binding/napi-v6-linux-glibc-x64/node_sqlite3.node" (not found)
npm error node-pre-gyp http GET https://github.com/TryGhost/node-sqlite3/releases/download/v5.1.4/napi-v6-linux-glibc-x64.tar.gz
npm error node-pre-gyp info install unpacking napi-v6-linux-glibc-x64/node_sqlite3.node
npm error node-pre-gyp ERR! Completion callback never invoked! 
npm error node-pre-gyp ERR! System Linux 6.8.12-2-pve
npm error node-pre-gyp ERR! command "/usr/bin/node" "/opt/wikijs/node_modules/sqlite3/node_modules/.bin/node-pre-gyp" "install" "--fallback-to-build"
npm error node-pre-gyp ERR! cwd /opt/wikijs/node_modules/sqlite3
npm error node-pre-gyp ERR! node -v v20.18.0
npm error node-pre-gyp ERR! node-pre-gyp -v v1.0.11
npm notice
npm notice New minor version of npm available! 10.8.2 -> 10.9.0
npm notice Changelog: https://github.com/npm/cli/releases/tag/v10.9.0
npm notice To update run: npm install -g npm@10.9.0
npm notice
npm error A complete log of this run can be found in: /root/.npm/_logs/2024-10-31T13_42_42_180Z-debug-0.log

[ERROR] in line 54: exit code 0: while executing command $STD npm rebuild sqlite3`

**Comment by Mellowlynx** [comment, marked as answer]
Runs fine here:
```
 _       ___ __   _     _     
| |     / (_) /__(_)   (_)____
| | /| / / / //_/ /   / / ___/
| |/ |/ / / ,< / /   / (__  ) 
|__/|__/_/_/|_/_(_)_/ /____/  
                 /___/      
 
Using Default Settings
Using Distribution: debian
Using debian Version: 12
Using Container Type: 1
Using Root Password: Automatic Login
Using Container ID: 211
Using Hostname: wikijs
Using Disk Size: 3GB
Allocated Cores 1
Allocated Ram 512
Using Bridge: vmbr0
Using Static IP Address: dhcp
Using Gateway IP Address: Default
Using Apt-Cacher IP Address: Default
Disable IPv6: No
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server Address: Host
Using MAC Address: Default
Using VLAN Tag: Default
Enable Root SSH Access: No
Enable Verbose Mode: No
Creating a Wikijs LXC using the above default settings
 ✓ Using local for Template Storage.
 ✓ Using local-lvm for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 101 was successfully created.
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: 10.0.0.211 
 ✓ IPv4 Internet Connected
 ✗ IPv6 Internet Not Connected
 ✓ DNS Resolved github.com to 140.82.121.4
 ✓ Updated Container OS
 ✓ Installed Dependencies
 ✓ Set up Node.js Repository
 ✓ Installed Node.js
 ✓ Installed Wiki.js
 ✓ Created Service
 ✓ Customized Container
 ✓ Cleaned
 ✓ Completed Successfully!

Wikijs should be reachable by going to the following URL.
         http://10.0.0.211:3000 
```
